code/modules/mob/mob_defines.dm

```diff
@@ -107,6 +107,7 @@
 
 	var/datum/dna/dna = null//Carbon
 	var/radiation = 0 //Carbon
+	var/cultslurring = 0 
```

Comment:
It seems like another candidate to put in the `status_procs.dm` file - at least, that way, it shall behave consistently like other status effects, and be readily isolated and removed later if needed. This seems to have been added in order to add a "flavorful" temporary speech impediment - hooking into the status effect system does not seem like a good solution to this. However, it should do for the moment, at least until speech impediment filters can be separated from the genetics system. Nonetheless, omitting this system would be simple, and not cause much impact on the overall PR.